src/common/mfu_io.c

```diff
@@ -609,6 +669,8 @@ void daos_open(const char* file, int flags, mode_t mode, mfu_file_t* mfu_file)
 
     mfu_free(&name);
     mfu_free(&dir_name);
```

Comment:
Oh, we should have to free up these strings under different error cases.

Reply:
I think this should be fine, no? The error cases fall through to the `mfu_free`s, except the error case before `name` and `dir_name` are declared.

Reply:
Yes, you're right.  Sorry, I was skimming too quickly and misreading those ``rc = -1`` statements in the errors as ``return -1``.

Good to rebase and merge.

Thanks!

Reply:
That's okay :)
Thanks for all the feedback!